### ISSUE TITLE
Quotation marks in the output string of the google translate extension appeared to be something like `&#x27;`, resulting to incorrect translation

### ISSUE THREAD

**Issue by Touch-Night** [oobabooga/text-generation-webui#3822]
### Describe the bug

`'`, `"`, and maybe some other punctuation are incorrectly translated due to Unicode Hex Character Code is used in the string instead of the coresponding punctuation.

### Is there an existing issue for this?

- [X] I have searched the existing issues

### Reproduction

Add some prints in the extension and you can see the issue is happening on untranslated output.

### Screenshot

![image](https://github.com/oobabooga/text-generation-webui/assets/102669562/618f2569-0755-4a3d-a7d9-8271a68162c1)

### Logs

```shell
No errors in command line
```

### System Info

```shell
System: Windows 10
GPU: Nvidia RTX 3050Ti Laptop (modified to 8GB VRAM)
```

**Comment by zencyon** [comment]
just pulled latest version and upgraded everything, getting this same behaviour as well
any ' and " are output as &#x27; and &quot; since update (running same model, same instruct

**Comment by Touch-Night** [comment]
> just pulled latest version and upgraded everything, getting this same behaviour as well any ' and " are output as '&#x27'; and '&quot' since update (running same model, same instruct

I did a quick fix in #3827 , you can try that.

**Comment by Touch-Night** [comment]
Close as fixed
